workers/data_refinery_workers/processors/utils.py

```diff
@@ -152,7 +152,7 @@ def end_job(job_context: Dict, abort=False):
         # S3-sync Computed Files
         for computed_file in job_context.get('computed_files', []):
             # Ensure even distribution across S3 servers
-            nonce = ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(8))
+            nonce = ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(24))
```

Comment:
Just curious why this change?

Reply:
3 million titles.. just wanted to expand the space